content/secure-remote-access/README.md

```diff
@@ -0,0 +1,77 @@
+# Secure Remote Access
+
+## Overview
+Instead of using Qidi Link to access your printer, this guide serves to help you
+set up a VPN mesh network that allows you to access your printer using various Klipper
+apps for mobile devices, using your trusted network as opposed to Qidi's.
+
+## Prerequisites
+* Comfortable with SSH access to your printer, executing commands, and editing files from CLI
+* Have a free Tailscale account - https://login.tailscale.com/start
+
+**NOTICE:**
+All of the below steps should be done from a laptop or computer that you use to print from regularly for simplicity. None of these commands need to be executed over your tailscale network. Unless specifically called out, when instructions call you to ssh to your printer, the expectation is that you are connecting over your LAN IP address.
+
+The only step that requires the VPN IP address or [magicdns](https://tailscale.com/kb/1081/magicdns) name is the last step and that is only for the mobile klipper client you installed to be configured.
+
+## 1. Setup Tailscale VPN Account
+Follow the onboarding guidelines from Tailscale to create and register your first device, preferably your phone since you are replacing Qidi Link! Instructions are [here](https://tailscale.com/kb/1017/install). Once you see your phone listed in the devices, move on to the next step!
+
+## 2. Optional - Disable qidi link
+Since we don't need this service running anymore, let's scavenge back those resources! From the Qidi Plus 4 screen, enable lan-only mode.
+
+## 3. Install Tailscale
+ 
+To install tailscale on your printer, first connect via ssh and run this single command:
+
+```
+curl -fsSL https://tailscale.com/install.sh | sh
+```
+
+Follow the onscreen instructions to complete the installation process.
+
+Once you complete the installation and receive confirmation of installation, head over to your tailscale.com account to confirm it! Click the admin console link in the top right of the tailscale site and look for your printer in the list of machines. 
+
+**Alternative Option - Subnet Router**
+
+While out of scope for this specific documented procedure, it's worth calling out that one can configure a [subnet router](https://tailscale.com/kb/1019/subnets) to avoid having to install the tailscale client on your printer. It allows for more flexibility but comes at the cost of complexity; ie dedicated device acting as the subnet router and configuring it as a gateway on devices that you want access to. This notice is to draw awareness to the end user that alernative installations exist. 
```

Comment:
Tailscale subnet router does not require configuration from the devices on that subnet at all. If you have a tailscale subnet router exposing `192.168.1.0/24` to tailscale, devices on the tailnet can access the subnet through tailscale, but devices on the subnet can't access the tailnet. It's essentially a split-vpn.